kujira/assetlist.json

```diff
@@ -1155,6 +1155,1134 @@
         "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/migaloo/images/rstk.png",
         "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/migaloo/images/rstk.svg"
       }
+    },
+    {
+        "description": "Ghost Vault USK",
+        "denom_units": [
+            {
+                "denom": "factory/kujira1w4yaama77v53fp0f9343t9w2f932z526vj970n2jv5055a7gt92sxgwypf/urcpt",
+                "exponent": 0
+            },
+            {
+                "denom": "xusk",
+                "exponent": 6
+            }
+        ],
+        "base": "factory/kujira1w4yaama77v53fp0f9343t9w2f932z526vj970n2jv5055a7gt92sxgwypf/urcpt",
+        "name": "Ghost Vault USK",
+        "display": "xusk",
+        "symbol": "xUSK",
+        "logo_URIs": {
+            "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/kujira/images/ghost/xusk.png"
+        },
+        "images": [
+            {
+                "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/kujira/images/ghost/xusk.png"
+            }
+        ]
+    },
+    {
+        "description": "Ghost Vault MNTA",
+        "denom_units": [
+            {
+                "denom": "factory/kujira1ya42knfcsvy6eztegsn3hz7zpjvhzn05ge85xa2dy2zrjeul9hnspp3c06/urcpt",
+                "exponent": 0
+            },
+            {
+                "denom": "xmnta",
+                "exponent": 6
+            }
+        ],
+        "base": "factory/kujira1ya42knfcsvy6eztegsn3hz7zpjvhzn05ge85xa2dy2zrjeul9hnspp3c06/urcpt",
+        "name": "Ghost Vault MNTA",
+        "display": "xmnta",
+        "symbol": "xMNTA",
+        "logo_URIs": {
+            "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/kujira/images/ghost/xmnta.png"
+        },
+        "images": [
+            {
+                "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/kujira/images/ghost/xmnta.png"
+            }
+        ]
+    },
+    {
+        "description": "Stride staked ATOM",
+        "denom_units": [
+            {
+                "denom": "ibc/0306D6B66EAA2EDBB7EAD23C0EC9DDFC69BB43E80B398035E90FBCFEF3FD1A87",
+                "exponent": 0
+            },
+            {
+                "denom": "statom",
+                "exponent": 6
+            }
+        ],
+        "base": "ibc/0306D6B66EAA2EDBB7EAD23C0EC9DDFC69BB43E80B398035E90FBCFEF3FD1A87",
+        "name": "stATOM",
+        "display": "statom",
+        "symbol": "stATOM",
+        "logo_URIs": {
+            "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/stride/images/statom.png",
+            "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/stride/images/statom.svg"
+        },
+        "images": [
+            {
+                "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/stride/images/statom.png",
```

Comment:
Please include image_sync for all images referencing images stored in another chain's /images/ directory.